### PR TITLE
addpkg(main/trzsz-ssh): 0.1.21

### PR DESCRIPTION
**Pull request by devome**

### CODE REVIEW

**Comment by TomJo2000** [comment]
So this is a self-described "drop in replacement for an OpenSSH client".
What exactly is the distinguishing feature then?

Just not having OpenSSH's `sshd` or `sftp` tied to it?

**Comment by TomJo2000** [comment]
I've taken a look at the source repository, and it looks like the 2.x releases have substantial differences from OpenSSH, such as a TUI interface.

I think version 2.1.3 would be more interesting for us to ship as a package than 0.1.21, but that's just my surface level assessment.

**Comment by TomJo2000** [comment]
Wait a minute I was looking at a completely different repo, also called tssh, also written in Go.
https://github.com/luanruisong/tssh

**Comment by devome** [comment]
@TomJo2000 I have change package name to `trzsz-ssh`, and the source code at https://github.com/trzsz/trzsz-ssh. The latest version is 0.1.21.  In order to unnecessary understanding of ambiguity, I changed the description.

**Comment by TomJo2000** [comment]
Please make sure to keep your commits squashed by the way.
For adding to a single commit you can use `git commit --amend`.
Since you already have 2 on your branch though, you'll need to squash those with `git rebase -i HEAD~2` first.
https://www.baeldung.com/ops/git-squash-commits#1-squash-the-last-x-commits

Since squashing or amending commits changes the git history you will need to force push any such changes.
e.g. `git push --force`,
or preferably `git push --force-with-lease=branchname`
to make sure you aren't clobbering any refs you haven't fetched locally yet.

**Comment by devome** [comment]
@TomJo2000 It‘s OK now.

**Comment by TomJo2000** [comment]
It probably makes sense to add `trzsz` itself in the same PR.
I'll write a package build for it and add it to this PR shortly.

**Comment by TomJo2000** [comment]
Hmm since they are so closely tied to each other I think it makes sense for both packages to be added in the same commit.

**Comment by devome** [comment]
@TomJo2000 I'm not sure if you're talking about [trzsz-go](https://github.com/termux/termux-packages/tree/master/packages/trzsz-go),  I think it is already in this repo.

**Comment by TomJo2000** [comment]
Wait... We already have a `trzsz-go` package.
https://github.com/termux/termux-packages/blob/6d9582be4b1e4f3a9c0e00fdb08ba20a625e1dec/packages/trzsz-go/build.sh#L1-L19

**Review by TomJo2000 — APPROVED** [review]

File: packages/trzsz-ssh/build.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://trzsz.github.io/ssh
+TERMUX_PKG_DESCRIPTION="An openssh client alternative"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.1.21"
+TERMUX_PKG_SRCURL=https://github.com/trzsz/trzsz-ssh/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=598d749e50af298700bfaa416690383bb16b9fc3d15ec66951857323726e2124
+TERMUX_PKG_AUTO_UPDATE=true
```
Comment (TomJo2000) [inline packages/trzsz-ssh/build.sh:8]: You can add `trzsz-go` as a recommended package for `trzsz-ssh`.
```suggestion
TERMUX_PKG_RECOMMENDS='trzsz-go'
TERMUX_PKG_AUTO_UPDATE=true
```

And likewise in `packages/trzsz-go/build.sh`
```bash
TERMUX_PKG_REVISION=1
TERMUX_PKG_RECOMMENDS='trzsz-ssh'
```

Comment (TomJo2000) [reply packages/trzsz-ssh/build.sh:8]: Eh quick fix, let me push a commit revision quick.

**Comment by devome** [comment]
@TomJo2000 TERMUX_PKG_RECOMMENDS has been added.

**Comment by TomJo2000** [comment]
> @TomJo2000 TERMUX_PKG_RECOMMENDS has been added.

Thanks, saves me the trouble.

**Comment by TomJo2000** [comment]
I've updated `trzsz-go` as well.
You can pull down those changes with `git -c pull.rebase=true pull`.

<h1><em></em></h1>  

As a note for future PRs, I'd recommend creating a separate feature branch on your fork.
This will let you keep the `master` branch updated without needing to constantly rebase your PR against every change.

**Comment by TomJo2000** [comment]
Speaking of clobbering, I just noticed I accidentally overwrote the `TERMUX_PKG_RECOMMENDS='trzsz-go'` in `trzsz-ssh`.

**Pull request merged**